### PR TITLE
Add error badge to FormError

### PR DESCRIPTION
**Pull request by msdehoward**

### CODE REVIEW

File: lib/components/Field/Field.module.scss [hunk 1]
```diff
@@ -76,9 +76,12 @@ $line-height: 5*$grid-size;
 .field-error {
     @include md-box(block, ellipsis);
     height: $form-error-height;
-    width: 100%;
+    width: fit-content;
```
Comment (PatoBeltran) [inline lib/components/Field/Field.module.scss]: why?, if the error is huge, will this make the width bigger than the field?

File: lib/components/Field/FormError.tsx [hunk 2]
```diff
@@ -30,12 +33,18 @@ export const FormError: React.StatelessComponent<FormErrorProps> = (props: FormE
...
+                hidden: props.hidden
             }, props.className)}
             title={props.title}
             attr={props.attr.container}
         >
-            {props.children}
+            {props.children 
+                ? <>
+                    {!props.hideIcon && <Icon icon='errorBadge' size={IconSize.small} className={css('error-badge')} />}
+                    <span className={css('inline-text-overflow')}>{props.children}</span>
```
Comment (PatoBeltran) [inline lib/components/Field/FormError.tsx]: why add `inline-text-overflow` was this behavior already there? Is anyone else setting this class to the error?

File: lib/components/Field/Field.module.scss (hunk 1 above)
Comment (msdehoward) [reply lib/components/Field/Field.module.scss]: reverted

File: lib/components/Field/FormError.tsx (hunk 2 above)
Comment (msdehoward) [reply lib/components/Field/FormError.tsx]: the behavior was not there before but it should have been. without this addition, the text is not properly clipped if it overflows

**Pull request merged**